charts/rancher-cis-benchmark/0.2.0/values.yaml

```diff
@@ -30,7 +30,7 @@ image:
     tag: v0.1.14
     pullPolicy: Always
   sonobuoy:
-    repository: rancher/sonobuoy-sonobuoy
+    repository: rancher/mirrored-sonobuoy-sonobuoy
```

Comment:
Is there a newer version of this ? See scanning issue.

Reply:
I see this mirrored already - https://hub.docker.com/r/rancher/mirrored-sonobuoy-sonobuoy 
Do you mean pulling in updated upstream sonobuoy?

Reply:
Talked offline, we are going to check the upstream docs to see if we can upgrade this to newer sonobuoy.

Reply:
I created this ticket  https://github.com/rancher/rancher/issues/33335 to test CIS with upstream newer sonobuoy to make sure there is no breaking change, can we push this fix to introduce the mirroring fixes with the existing version @cbron cc @deniseschannon